peachjam/adapters/adapters.py

```diff
@@ -0,0 +1,150 @@
+import logging
+from tempfile import NamedTemporaryFile
+
+import magic
+import requests
+from dateutil import parser
+from django.core.files import File
+
+from peachjam.plugins import plugins
+
+logger = logging.getLogger(__name__)
+
+
+class Adapter:
+    def __init__(self, settings):
+        self.client = requests.session()
+        self.client.headers.update(
+            {
+                "Authorization": f"Token {settings['token']}",
+            }
+        )
+        self.url = settings["url"]

```

Comment:
We can't make any assumptions about what the adapter wants to do with the settings.

```suggestion
    def __init__(self, settings):
        self.settings = settings
```